packages/carbon-graphs/tests/unit/controls/Timeline/TimelineLoad-spec.js

```diff
@@ -756,4 +762,33 @@ describe('Timeline - Load', () => {
       ).toEqual(inputPrimary.key);
     });
   });
+  describe('When timeline is created as second graph in page', () => {
+    it('should render graph from right point', () => {
```

Comment:
nit: `should render graph from the right side of the container`

Reply:
It is the correct position, Andy. Fixed [here](https://github.com/cerner/terra-graphs/pull/91/commits/c80f2946bf5a8d9dcb35cff6acf77e7f3cd46d8d#diff-468a3e72133761d3dc61f876f1c9b2a5c228031c41b11906708ce33d186a957bR766).